.github/workflows/ci.yml

```diff
@@ -34,22 +34,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - run: rustup target add thumbv7m-none-eabi
       - name: Install cargo-hack
         uses: taiki-e/install-action@cargo-hack
       - run: cargo hack build --rust-version
-
-  no-std:
-    strategy:
-      matrix:
-        rust:
-          # This is the minimum supported Rust version for "no_std"
-          - 1.36.0
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - name: Install Rust
-        run: rustup update ${{ matrix.rust }} && rustup default ${{ matrix.rust }}
-      - run: rustup target add thumbv7m-none-eabi
       - run: cargo build --no-default-features --target thumbv7m-none-eabi
```

Comment:
This is run for the default rustc version, so no longer work as MSRV check. Please use cargo-hack's `--rust-version` flag like the above command and the command I mentioned in the above review (it should also eliminate the need for manual `rustup target add`.).

Reply:
Ops. Should be fixed now.